File: app/src/main/java/omniex/nl/omniex/ui/app/profile/order/ProfileOrderView.java
package omniex.nl.omniex.ui.app.profile.order;

import java.util.List;

import nl.omniex.omniexshopping.data.model.order.OrderStatus;
import nl.omniex.omniexshopping.ui.base.BaseView;

public interface ProfileOrderView extends BaseView {

    void onOrderStatusesFetched(List<OrderStatus> orderStatusList);
}

File: app/src/main/java/omniex/nl/omniex/ui/app/profile/order/ProfileOrderView.kt
package omniex.nl.omniex.ui.app.profile.order

import omniex.nl.omniex.data.model.order.OrderStatus
import omniex.nl.omniex.ui.base.BaseView


interface ProfileOrderView : BaseView {

    fun onOrderStatusesFetched(orderStatusList: List<OrderStatus>)
}
